app/Models/Context.php

```diff
@@ -97,7 +97,7 @@ public static function initUser($username = '', $userMustExist = true) {
 					FRESHRSS_PATH . '/config-user.default.php',
 					FreshRSS_Context::$system_conf->configurationSetter());
 
-				FreshRSS_Context::currentUser($username);
+				Minz_Session::_param('currentUser', $username);
```

Comment:
Maybe, now that we are at it, it would make sense to introduce the corresponding `FreshRSS_Context::setCurrentUser()` method. It would also be easier for type hints.

Reply:
done